src/Service/PagePurger.php

```diff
@@ -43,6 +44,24 @@ public function purge( Page $page ) {
 	}
 
 	/**
+	 * @since 0.x
```

Comment:
Adding @since 0.7 should be fine for now (Before releases I review everything anyway).
Semver is indeed used, but I have only been doing 0.x as in theory nothing would be added in a 0.x.>1 release.
Either 0.7 or 0.7.0 are both fine for me!